shared/actions/settings.js

```diff
@@ -421,6 +421,36 @@ const setLockdownMode = (state: TypedState, action: SettingsGen.OnChangeLockdown
       return SettingsGen.createLoadLockdownMode()
     })
 
+const unfurlSettingsRefresh = (state: TypedState, action: SettingsGen.UnfurlSettingsRefreshPayload) =>
+  state.config.loggedIn &&
+  ChatTypes.localGetUnfurlSettingsRpcPromise(undefined, Constants.waitingKey)
```

Comment:
Totally not your fault, but having a waitingKey per-RPC makes more sense: switching to `Constants.unfurlWaitingKey` for these would be much better.